adopt-a-dog/blog.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="ZXX" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Blog</title>
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.2/css/all.css" integrity="sha384-/rXc/GQVaYpyDdyxK+ecHPVYJSN9bmVFBvjA/9eOB+pb3F2w2N6fc5qB9Ew5yIns" crossorigin="anonymous">
+      <link href="https://fonts.googleapis.com/css?family=Indie+Flower" rel="stylesheet">
+      <link href="https://fonts.googleapis.com/css?family=PT+Sans" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+    <img class="logo" src="adoptadog-logo.svg" alt="Adopt a dog logo">
+    <img id="cart-icon" src="cart-icon.svg" alt="cart-icon">
+  </header>
+    <nav>
+        <a href="index.html">Home </a>  |
+        <a href="dogs.html">Dogs</a>   |
+        <a href="blog.html">Blog</a>
+    </nav>
+    <h1>Adopt A Dog Blog</h1>
+    <div class="blog-card">
+      <img class="Eric" src="images/patrick-hendry-221863-unsplash.JPG" alt="A woman and a Dog">
```

Comment:
this class name probably isn't the best for this kind of thing. I'd recommend using something like "blog-image" for these purposes.